test/tests_for_sdkmr/ee2_SDKMethodRunner_test_EE2Runjob_test.py

```diff
@@ -237,6 +237,43 @@ def test_run_job(self, rq_mock, condor_mock):
         job_id = runner.run_job(params=job)
         print(f"Job id is {job_id} ")
 
+    @requests_mock.Mocker()
+    @patch("lib.execution_engine2.utils.Condor.Condor", autospec=True)
+    def test_run_job_batch(self, rq_mock, condor_mock):
+        rq_mock.add_matcher(
+            run_job_adapter(
+                ws_perms_info={"user_id": self.user_id, "ws_perms": {self.ws_id: "a"}}
+            )
+        )
+        runner = self.getRunner()
+        runner.get_condor = MagicMock(return_value=condor_mock)
+        job = get_example_job(user=self.user_id, wsid=self.ws_id).to_mongo().to_dict()
+        job["method"] = job["job_input"]["app_id"]
+        job["app_id"] = job["job_input"]["app_id"]
+        job["service_ver"] = job["job_input"]["service_ver"]
+
+        si = SubmissionInfo(clusterid="test", submit=job, error=None)
+        condor_mock.run_job = MagicMock(return_value=si)
+        condor_mock.extract_resources = MagicMock(return_value=self.cr)
+
+        jobs = [job, job, job]
+        job_ids = runner.run_job_batch(params=jobs, batch_params={"wsid": self.ws_id})
+        print(f"Job ids are {job_ids} ")
```

Comment:
What are you expecting `job_ids` to look like? Can you write a test, instead of printing them out?

Reply:
I didn't expect them to look any different than regular mongo _ids. It was helpful to see these when I was writing the tests. I can delete this print statement.